backend/src/main/resources/static/js/getLocation.js

```
@@ -0,0 +1,34 @@
+          resolve (await getIpLocation());
+        }catch(error){
+          reject(new Error("Location with IP address failed: " + error.message));
+        }
+      },
+      {
+        enableHighAccuracy: true,
+        timeout: 5000,
+        maximumAge: 0,
+      }
+    )
+  });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)